ngraph/test/visitors/op/convolution_backprop.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+
+#include "ngraph/ngraph.hpp"
+#include "ngraph/op/util/attr_types.hpp"
+#include "ngraph/opsets/opset1.hpp"
+#include "ngraph/opsets/opset3.hpp"
+#include "ngraph/opsets/opset4.hpp"
+#include "ngraph/opsets/opset5.hpp"
+
+#include "util/visitor.hpp"
+
+using namespace std;
+using namespace ngraph;
+using ngraph::test::NodeBuilder;
+using ngraph::test::ValueMap;
+
+TEST(attributes, convolution_backprop_op)
```

Comment:
This looks good overall, I have just one thing I wanted to let you know:
In this [PR](https://github.com/openvinotoolkit/openvino/pull/5430) We merged the visitor attribute tests for ops with no attributes, using ValueMap stored in the visitor, I'm thinking that It would be possible to compare the ValueMaps in some way, and this would cover comparing all the attributes, the count etc.

Of course this is not a must (as it would require some additional work), but even if you decide to stick with this approach, I'd like to see the attribute count comparison in the tests you're adding here- so we know in the future there isn't some mysterious attribute added, breaking backwards compatibility.

Reply:
Attribute counter check is added.